tests/system/system_test.go

```diff
@@ -22,15 +22,22 @@ func NewSystemTest(t *testing.T, dir string, vars map[string]string) *SystemTest
 	vars["endpoint"] = os.Getenv(config.ENVVAR_ENDPOINT)
 	vars["token"] = os.Getenv(config.ENVVAR_TOKEN)
 
+	t.Logf("[DEBUG] endpoint: %s", vars["endpoint"])
+	t.Logf("[DEBUG] token: %s", vars["token"])
+
 	tfContext := tftest.NewTestContext(t,
 		tftest.Dir(dir),
 		tftest.Vars(vars),
 		tftest.DryRun(*dry))
 
 	layer0 := clients.NewLayer0TestClient(t, vars["endpoint"], vars["token"])
 
-	// download modules using terraform get
-	tfContext.Terraformf("get")
+	// download modules using terraform init
+	stdoutStderr, err := tfContext.Terraformf("init")
```

Comment:
I think we should just use `tfContext.Init()` here

Reply:
✅